kaiax/gasless/impl/execution.go

```diff
@@ -24,5 +24,5 @@ import (
 var _ kaiax.ExecutionModule = (*GaslessModule)(nil)
 
 func (g *GaslessModule) PostInsertBlock(block *types.Block) error {
-	return g.updateAddresses(block.Number())
+	return g.updateAddresses()
```

Comment:
I have a question. Why you used current.block number instead of block.Number()?

Reply:
fixed at https://github.com/kaiachain/kaia/pull/297/commits/c4825b577dbaaf0f16c2e4bc559f3b98f4a66242